v3/src/components/case-table/case-table.scss

```diff
@@ -2,7 +2,7 @@
 @use "./case-table-shared.scss" as shared;
 @use "../case-tile-common/case-tile-vars" as caseTileVars;
 
-$table-body-font-size: 8pt;
+$table-body-font-size: 12px;
```

Comment:
As discussed in the meeting yesterday, the font size changes are tied to a corresponding row height change, which we decided was not part of this PR. The new cell font size is clearly too large for the existing row height, so we should use something smaller (e.g. 10px or 11px) for now and include increasing the font size in the new row height story. I believe @mtirenin was tasked with choosing an appropriate font size for the current row height in yesterday's meeting.